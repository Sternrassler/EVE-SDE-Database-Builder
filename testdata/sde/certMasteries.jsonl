{"typeID":587,"masteryLevel":0,"certID":1}
{"typeID":587,"masteryLevel":1,"certID":2}
{"typeID":588,"masteryLevel":0,"certID":3}
