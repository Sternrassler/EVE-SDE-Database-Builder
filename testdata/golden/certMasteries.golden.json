[
  {
    "typeID": 587,
    "masteryLevel": 0,
    "certificateID": null
  },
  {
    "typeID": 587,
    "masteryLevel": 1,
    "certificateID": null
  },
  {
    "typeID": 588,
    "masteryLevel": 0,
    "certificateID": null
  }
]